examples/feeds/update_flights_feed_item_string_attribute_value.py

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates a feed item attribute value in a flights feed.
+
+To create a flights feed, run the AddFlightsFeed example. This example is
```

Comment:
Unfortunately this example does not yet exist in Python. Could you add it to this PR?

Reply:
It's under `examples/remarketing`, I'll change this to make that more clear.